flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumerTest.java

```diff
@@ -87,18 +87,6 @@ public void testUnrecognizableAwsRegionInConfig() {
 	}
 
 	@Test
-	public void testCredentialProviderTypeDefaultToBasicButNoCredentialsSetInConfig() {
```

Comment:
Makes sense, just realized that we don't really have unit tests for `AWSUtil` right now, and the removed test simply corresponds to the reduced behaviour of `validateAwsConfiguration()`. Thanks for pointing this out!